### ISSUE TITLE
add Ghostty

### ISSUE THREAD

**Issue by davay** [godbout/kindaVim.docs#259]
[Ghostty](https://ghostty.org) is a newly-released terminal, the wizard should recommend handling it the same way as other terminals -- set it to the off family

**Comment by godbout** [comment]
yeah so it (finally!) came out yesterday. will do a bit of extra testing for Wooshy first, and then i'll add it to kV for the next release. should happen within the next couple of days.

**Comment by davay** [comment]
No rush! Just thought of making this for docs purposes, thank you 😊

**Comment by godbout** [comment]
definitely. thank you! a release is needed soon because i've been ticking off a big amount of todos already. so time to release and move on to the next things. will add no worries.

**Comment by godbout** [comment]
added in kV70: https://github.com/godbout/kindaVim.docs/releases/tag/70
